wp-content/themes/midwestenergynews/partials/content-single-classic.php

```diff
@@ -21,7 +21,7 @@
 
 	<div class="row story-row">
 		<div class="span3">
-			<h5 class="byline"><?php largo_byline(); ?></h5>
+			<h5 class="byline"><?php largo_byline( true, false, $post->ID ); ?></h5>

```

Comment:
@benlk Since you're checking if `(is_object($post)) {` in `largo_byline` should this use `$post` as the argument instead of the ID?